tests/ternary-object-expressions/with-balanced-formatting/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -96,13 +96,13 @@ let isSpace = false;
 
 const dress = isSpace
 	? {
-		spaceSuit: 3,
-		oxygenCylinders: 6
-	}
+			spaceSuit: 3,
+			oxygenCylinders: 6
+	  }
 	: {
-		shirts: 3,
-		paints: 3
-	};
+			shirts: 3,
+			paints: 3
+	  };
```

Comment:
Here is the diff from my terminal with display tab width of 8:

```diff
@@ -96,13 +96,13 @@ let isSpace = false;
 
 const dress = isSpace
        ? {
-               spaceSuit: 3,
-               oxygenCylinders: 6
-       }
+                       spaceSuit: 3,
+                       oxygenCylinders: 6
+         }
        : {
-               shirts: 3,
-               paints: 3
-       };
+                       shirts: 3,
+                       paints: 3
+         };
 
 ================================================================================
 `;
```

I think a workaround like using a "--no-ternary-inner-indent" option as discussed in #585 is clearly needed when using the balanced ternary formatting option together with `--use-tabs`.